tests/test_query.rs

```diff
@@ -0,0 +1,198 @@
+use actix_web::{
+    test::{self, TestRequest},
+    web,
+};
+#[cfg(test)]
+use boom_api::{
+    api::{query, query::build_options},
+    models::query_models::{Query, QueryBody, QueryKwargs, Unit},
+};
+use mongodb::{
+    bson::{doc, Document},
+    options::FindOptions,
+    Client,
+};
+
+// TODO: put in conf
+const DB_NAME: &str = "boom";
```

Comment:
the unit tests should run on a different DB, say "boom_test", just like we do for `boom`'s unit tests